src/rules/jsx-one-expression-per-line.coffee

```diff
@@ -77,8 +77,7 @@ module.exports =
         countNewLinesAfterContent = 0
 
         if child.type in ['Literal', 'JSXText']
-          # TODO: this is only necessary b/c JSXFragments aren't currently getting transformed Babel -> espree
```

Comment:
I'd run into this before but I guess hadn't known how to get JSX fragments to be transformed by the version of `babel-eslint` we use (which is old)